exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,15 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if (mark >= 80)
+    return "A";
+  else if (mark < 80 && mark > 60)
+    return "B";
+  else if (mark <= 60 && mark >=50)
+    return "C";
+  else return "F";
+}
```

Comment:
This works, but you could make the code simpler.  Your code will exit and return "A"  in line 13 if 
mark >=80
 So you don't need to do the check 
mark < 80 
in line 14 - if must be less than 80 or your code would already have exited the function. This line could be 
else if (mark > 60)
and it will still work.
Similarly, you can simplify line 16

Reply:
Thank you Karen, I have done silly mistake here, making it doing same work twice. I'm going to change it.